core/videolib/convertor.py

```diff
@@ -25,14 +26,15 @@ def convert_video(video_input: str, video_output: str) -> None:
     video_output : str
         The output file name
     """
-
-    flags = ["ffmpeg", "-i", f"{video_input}.ts", "-acodec", "copy", "-vcodec", "copy", video_output]
+    flags = ["ffmpeg", "-i", f"{video_input}.ts", "-acodec"
+        , "copy", "-vcodec", "copy", video_output]
     subprocess.Popen(flags, stdout=subprocess.DEVNULL).wait()
     os.unlink(f"{video_input}.ts")
 
 
 def get_ts_start_time(file_path: str) -> float:
-    """Parse the downloaded file
+    """Search For ts file start time.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)